content/learn/contribute/helping-out/creating-examples.md

```diff
@@ -29,19 +71,52 @@ Please adhere to the following guidelines when creating or updating an example.
 ### Organization
 
 1. Examples should live in an appropriate subfolder of `/examples`.
-2. Examples should be a single file if possible.
-3. Assets live in `/assets`. Try to avoid adding new assets unless strictly necessary to keep the repository small. Don't add "large" asset files.
+   1. API examples live in `examples/api`, then subdivided by engine category.
+   2. Usage examples live in `examples/usage`, then subdivided by domain.
+   3. Game examples live in `examples/games`, with each game example getting its own dedicated subfolder.
+   4. Stress tests live in the root level `stress-tests` folder.
+   5. Testbeds live in the root level `tests` folder, subdivided by domain (e.g. `3d`).
+2. Each example should consist of exactly one `.rs` file, except where assets or shaders are required.
+   1. Game examples are an exception to this, and should be structured as self-contained small projects with a realistic file and module structure.
+3. Assets live in `/assets`, organized by kind, rather than by example.
 4. Each example should try to follow this order:
    1. Imports
    2. A `fn main()` block
    3. Example logic
 5. Try to structure app / plugin construction in the same fashion as the actual code.
 6. Examples should typically not have tests, as those are not directly reusable by the Bevy user.
+7. Examples should not share common "utility" logic: if you feel this is warranted, improve the engine instead!
+   1. Similarly, try to avoid incidental complexity and abstractions. Find a simpler way for now, and work to create these tools and resolve this tension in the engine itself.
```

Comment:
I'd want to contribs to file issues for pain points like these which they encounter during example development.